python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -736,3 +736,20 @@ def __firstRowPlug( plugs ) :
 
 		return None
 
+# Ensures navigation key presses aren't stolen by any application-level actions.
+class _NavigableTable( _TableView ) :
```

Comment:
Might this behaviour be better off in `_TableView` itself?

Reply:
I was mulling that, but then found it hard to rationalise _which_ key presses we should preserve. Happy to move it if you want to work out a suitable generalisation? The `HierarchyView`, for example, doesn't need this behaviour iirc.